### PR TITLE
Adjust for new frontend

### PR DESCRIPTION
**Pull request by markbreedlove**
Migrate our frontend architecture based on the public rollout of https://github.com/dpla/dpla-frontend

Also clean up various things that aren't directly related to the relaunch.

The merge of this PR will be tagged as a major-version release.

* Change development VMs
* Remove `wordpress`, `frontend`, `exhibitions`, and `mysql` roles
* Remove obsolete inventory
* Remove PostgreSQL `pg_maintenance*` playbooks. We only use the `postgresql` role in development now. That's actually been the case for quite a while.
* Remove `central_all.yml`, `production_all.yml`, and `staging_all.yml` playbooks. These have been moved to another repository.
* Adjust the application of the `site_proxy` tag
* Restrict the `swapper.yml` playbook to an `ec2_swapper` inventory group if it exists.

### CODE REVIEW

**Comment by markbreedlove** [comment]
Please hold, changes coming ...

**Comment by markbreedlove** [comment]
Should be OK to review again. Please note new instructions in the READMEs.

File: ansible/playbooks/init_index_and_repos.yml [hunk 1]
```diff
@@ -41,36 +36,13 @@
           force_basic_auth: yes
           status_code: "200,404"
       with_items:
-        - dpla
-        - dashboard
```
Comment (moltude) [inline ansible/playbooks/init_index_and_repos.yml]: Why is `dashboard` being removed here? I undertstand that `dpla` and `bulk_download` are no longer used for ingestion1 but `dashboard` is still reqired for job tracking.

**Comment by markbreedlove** [comment]
@moltude I've added the `dashboard` database back, per your suggestion.

**Pull request merged**